models/gamesM.js

```diff
@@ -21,3 +21,32 @@ exports.getSingleReviewByID = (reviewID) => {
       return Promise.reject(err);
     });
 };
+
+exports.addReviewVotes = (reviewID, voteInc) => {
+  if (voteInc < 0) {
+    return Promise.reject({ status: 400, msg: "Votes must be positive" });
+  }
+
+  return db
+    .query(`SELECT * FROM reviews WHERE review_id=$1`, [reviewID])
+    .then((data) => {
+      let currentVotes = data.rows[0].votes;
+      let newVotes = currentVotes + voteInc;
+      return db
+        .query(`UPDATE reviews SET votes = $1 WHERE review_id=$2`, [
+          newVotes,
+          reviewID,
+        ])
```

Comment:
Whew, this is a little overcomplicated - not wrong, but if you want to refactor it, you could get away with just including SET votes = votes + $1 in your SQL Statement. Nothing that you need to change, just something to keep in mind to save you some time in the future :)

Reply:
Oh yeah, that is much neater. Thanks :D